src/material-experimental/mdc-chips/chip-grid.ts

```diff
@@ -379,6 +379,10 @@ export class MatChipGrid
    */
   setDescribedByIds(ids: string[]) {
     this._ariaDescribedby = ids.join(' ');
+
```

Comment:
I'm wondering whether we should be setting `aria-describedby` on the chip grid at all, considering that it never receives focus.

Reply:
That's a good point.

I don't see any harm in keeping it on the chip grid itself, but I also don't see any harm in removing it either.

The only (extremely minor) concern I have is if the focus management of the grid instance changes to focus the grid element itself at some point.  That's not the case here, though, and I don't really think it's a big deal if we remove it.

All of that is to say:  I can remove the attribute from the grid if you'd like, just let me know!

Reply:
I think that we should get rid of it, otherwise it's misleading.

Reply:
Works for me!  We still need to keep the property present on the control to satisfy the MatFormFieldControl interface, but we can at least remove the property binding.

Reply:
Actually maybe not lemme dig further into that.

Reply:
It's defined on MatChipSet, which means we'll need to keep the property itself but can remove the binding.